wemake_python_styleguide/logic/tree/variables.py

```diff
@@ -54,3 +54,17 @@ def _is_valid_single(node: _VarDefinition) -> bool:
         isinstance(node, ast.Name) or
         isinstance(node, ast.Starred) and isinstance(node.value, ast.Name)
     )
+
+
+def is_getting_first_element_by_unpacking(targets: List[ast.expr]) -> bool:
+    """Checks if unpacking targets used to get first element."""
+    return (
+        len(targets) == 2 and
+        isinstance(targets[0], ast.Name) and
+        isinstance(targets[1], ast.Starred) and
+        _is_unused_variable_name(targets[1].value)
```

Comment:
Let's refactor this part to use: https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/logic/naming/access.py#L9

Reply:
Oh, sorry. Let's use `is_protected` then

Reply:
This does not work too:

```
>>> is_protected('_')
False

>>> is_protected('__')
False
```

While `access.is_unused(node.id) or access.is_protected(node.id)` works.

But for me such code looks misleading. Would you consider adding different "weak" kind of `is_unused` in naming.access?

Reply:
Ok, then let's create a new function `access.looks_like_unused()`

Reply:
Done.